project-reports/2023/2023-Q1-Hyperledger-Indy.md

```diff
@@ -0,0 +1,162 @@
+will simplify the transition to "did:indy".
+
+[anoncreds-rs]: https://github.com/hyperledger/anoncreds-rs
+
+Per the [Indy Activity Dashboard (2022-10 to 2022-12)], there were 100 commits
+(almost double from last quarter) from 13 contributors (up one).
+
+[Indy Activity Dashboard (2022-10 to 2022-12)]: https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Findy/dashboard;subTab=technical?time=%7B%22from%22:%222022-10-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222022-12-31T07:00:00.000Z%22%7D
+
+## Questions/Issues for the TSC
+
+### Issues from previous reports
+
+#### Build Pipelines
+
+**Update** : The Ubuntu upgrade is complete, producing releases and
+compatibility with the Ubuntu 16.04 instance has been tested. To be removed from
+the next report.
+
+#### **Diversity of Contributor Community**
+
+**No Update** : Little change this quarter in contributor community. Lots
+of interest, but core maintainers continue to do most of the work.
+
+## Releases
+
+- indy-node (Ubuntu 20.04) – v1.13.2-rc4
+
+## Overall Activity in the Past Quarter
+
+In the past quarter (as in the previous quarter), ledger code
+development focused on code management – upgrading the Indy Node and
+Plenum CI/CD pipeline and upgrading Indy Node to run on Ubuntu 20.04.
+A particular focus was on the [indy-test-automation] pipeline, and
+enabling its use for both Indy Node releases and for the downstream
+Sovrin Node releases.
+
+Much activity occurred in extracting the AnonCreds Rust implementation
+from the [indy-shared-rs] repository to the standalone [anoncreds-rs]
+repository.
+
+## Current Plans
+
+With the new CI/CD Indy Node and Plenum pipelines complete, and the
+Ubuntu 20.04 indy-node upgrade available, the core maintainers are
+focused on their downstream releases.
+
+The Aries Frameworks are moving to [indy-vdr] and [aries-askar] and away from
+[indy-sdk]. As that happens, adjustments and additions will be made to the newer
+repositories. The high priority work on Hyperledger AnonCreds implementation
+will also drive that effort.
+
+A focus on pushing the "did:indy" DID Method is enabled by the upgrade of Indy
+Node to the new Ubuntu 20.04-based releases.
+
+## Maintainer Diversity
+
+The bi-weekly Indy Contributors call continues to be the medium by which
+maintainers coordinate work, discuss critical issues to the Indy
+codebase, and agree on larger changes. Topics and attendance has
+increased recently, with more and more interested parties showing up,
+and new contributors weighing in on the conversations.
+
+## Contributor Diversity
+
+The edX course about Indy, Aries, AnonCreds, and Ursa ([LFS172x - Introduction
+to Hyperledger Self Sovereign Identity Blockchain Solutions]) was updated
+recently, with the new version launched in January 2023.
+
+[LFS172x - Introduction to Hyperledger Self Sovereign Identity Blockchain Solutions]: https://www.edx.org/course/identity-in-hyperledger-aries-indy-and-ursa
```

Comment:
What does this have to do with Contributor Diversity? Is this intended for the Additional Information section?

Reply:
When I took over doing the Indy/Aries reports, that was the type of information in the section -- activities that led to Contributor Diversity.

I've moved it to the `Additional Information section` as requested and added a link to the activity dashboard in the contributor diversity. As I review the other Quarterly reports I'll look for other information to add next report.